DayrSuryan/DSEthiop10.xml

```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>Miscellaneous collection of hymns and prayers, Teaching of Mysteries</title>
+            <title>Miscellaneous collection of hymns and prayers, Doctrine of Mysteries</title>
```

Comment:
Please, let us live with "Teaching". It is the standard title. What is in the work is not a "doctrine".